README.md

```diff
@@ -307,14 +307,25 @@ Date: Mon, 27 Jun 2022 14:26:45 GMT
 As part of making this API performant, all reading endpoints support long-polling as an efficient alternative to regular (repeated) polling. Using this function requires the following parameters:
 
 - `wait_count`: The API call will block until this many results are available ...
-- `wait_time`: ... or this many milliseconds have passed, whichever comes first.
+- `wait_time`: ... or this many *milliseconds* have passed, whichever comes first.
 
 For example, retrieving a task's results:
 
 - `GET /v1/tasks/<task_id>/results` will return immediately with however many results are available,
 - `GET /v1/tasks/<task_id>/results?wait_count=5` will block forever until 5 results are available,
 - `GET /v1/tasks/<task_id>/results?wait_count=5&wait_time=30000` will block until 5 results are available or 30 seconds have passed (whichever comes first). In the latter case, HTTP code `206 (Partial Content)` is returned to indicate that the result is incomplete.
 
+### Experimental Server-sent Events (SSE) API
+
+To better support asnchronous use cases, such as federated search UIs, this development version supports a fist implementation of [Server-sent Events](https://www.rfc-editor.org/rfc/rfc8895.html#name-server-push-server-sent-eve) for *Result* retrieval. This allows Beam.Proxies to "subscribe" to tasks and get notifications for every new result without explicit polling. Please note, that this feature is experimentatal and subject to changes. In particular, the API endpoint stated bekow might change in the future.
+
+Method: `GET`  
+URL: `/v1/tasks/<task_id>/results/stream`  
+Parameters:
+
+ - The same parameters as for long-polling, i.e. `to`, `from`, `filter=todo`, `wait_count`, and `wait_time` are supported.
+
+
```

Comment:
This now takes the new `Accept` header, could you update it accordingly? See https://github.com/samply/beam/commit/403c5e0a876aa0f7a337a559af09f7803b633025

Reply:
Fixed in https://github.com/samply/beam/pull/105/commits/456faf7b8f71c6231e8620bd6d0f0c0db057c1b4